api/rebac/v1/check.proto

```diff
@@ -0,0 +1,31 @@
+syntax = "proto3";
+
+package api.rebac.v1;
+import "google/api/annotations.proto";
+option go_package = "ciam-rebac/api/rebac/v1;v1";
+option java_multiple_files = true;
+option java_package = "api.rebac.v1";
+
+service Check {
```

Comment:
I think this service should also include other lookups (subject lookups and reverse/object lookups). For generated clients it's just one class they can use rather than multiple.

So, maybe, `LookupService`?

I think we should also suffix each service with "Service" as from what I can tell this is common for gRPC services and makes the generated clients more readable.

Reply:
We can use name `lookup` for proto, but I don't such the convention "LookupService" is followed in Golang (more of Java)

Reply:
Agreed, I would stick to the convention followed with https://go-kratos.dev/en/docs/component/api
It adds a service suffix to internal/service: https://github.com/RedHatInsights/insights-rebac/blob/main/internal/service/relationships.go#L12

Reply:
Notes – choosing between:

```
service CheckService {...}
```

(which would only have check, maybe also content change check, and there would be another service for lookups)

vs

```
service PolicyService {...}
```

(which would have other lookups / expand as well – there may be a better name for it)

Pending input from early adopters.

Reply:
Note the "Service" suffix still important for client generation. Also see – https://buf.build/docs/lint/rules#service_suffix

Reply:
Please can you create a jira, we can take that in account for all API (service) follow the formate going forward?

Reply:
Will we go with the 3 way split for this PR: CheckService, LookupService and RelationTuplesService? (Or is that another PR?)